lib/dpp/DriveStateRepository.js

```diff
@@ -46,6 +49,12 @@ class DriveStateRepository {
     const transaction = this.getDBTransaction('identity');
 
     await this.identityRepository.store(identity, transaction);
+
+    const [firstPublicKey] = identity.getPublicKeys();
+
+    await this.publicKeyIdentityIdMapLevelDBRepository.store(
```

Comment:
We need to store it in a separate method `storePublicKeyIdentityId`, also we need `fetchPublicKeyIdentityId` for Identity Create Transition validation function.